src/main/resources/templates/surveyResponse.html

```diff
@@ -5,16 +5,21 @@
 <script src="/webjars/bootstrap/js/bootstrap.min.js"></script>
 <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/css/bootstrap.min.css" integrity="sha384-Vkoo8x4CGsO3+Hhxv8T/Q5PaXtkKtu6ug5TOeNV6gBiFeWPGFN9MuhOf23Q9Ifjh" crossorigin="anonymous">
 <script src="https://cdnjs.cloudflare.com/ajax/libs/Chart.js/2.9.3/Chart.bundle.js"></script>
-<script src="/js/responseSurvey.js"></script>
+<script th:if="${featureVar} == 'GRAPHICAL_RESPONSES'" src="/js/responseSurvey.js"></script>
 <link rel="stylesheet" type="text/css" href="/css/answerSurvey.css">
 <head>
     <title th:text="${name}"></title>
     <meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
 </head>
 <body>
+
     <nav class="navbar navbar-dark bg-dark">
         <a class="navbar-brand" href="#" th:text="'Responses for ' + ${name}"></a>
     </nav>
+    <div th:switch="${featureVar}">
+        <p hidden th:case="GRAPHICAL_RESPONSES" th:text="${featureVar}"></p>
+        <p hidden th:case="BASIC_RESPONSES" th:text="${featureVar}"></p>
```

Comment:
this looks like its doing the same thing for both cases?

Reply:
It'll have whatever featureVar is set to based on whether GRAPHICAL_RESPONSES is toggled or not. If its not set it's gonna show up as BASIC_RESPONSES, which isn't actually a feature but just some info to say the feature is not set currently with the page.

Reply:
Ah never mind, I see that it is unnecessary to perform this conditional. I will remove this.